templates/home/_fsf_flutter.html

```diff
@@ -0,0 +1,46 @@
+<div class="row u-hide u-no-padding" data-flow-details="flutter">
+  <div class="col-6">
+    <h4>Why are snaps good for Flutter projects?</h4>
+    <ul>
+      <li>Snaps are easy to discover and install. Millions of users can browse and install snaps graphically in the Snap Store or from the command-line.</li>
+      <li>Snaps install and run the same across Linux. They bundle the exact versions of your app’s dependencies.</li>
+      <li>Snaps automatically update to the latest version. Four times a day, users’ systems will check for new versions and upgrade in the background.</li>
+      <li>Upgrades are not disruptive. Because upgrades are not in-place, users can keep your app open as it’s upgraded in the background.</li>
+      <li>Upgrades are safe. If your app fails to upgrade, users automatically roll back to the previous revision.</li>
+    </ul>
+
+    <div class="p-flow-details__continue">
+      <p>In just a few steps, you’ll have an example Flutter app in the Snap Store.</p>
+      <a class="p-button--positive" href="first-snap/flutter">Continue &rsaquo;</a>
+    </div>
+  </div>
+
+  <div class="col-6">
+    <h4>Here's how <a href="https://snapcraft.io/flutter-gallery">flutter-gallery</a> defines snapcraft.yaml:</h4>
+    <div class ="p-show-more is-collapsed" data-js="js-show-more">
+      <pre class="p-code-yaml"><b>name</b>: flutter-gallery
+<b>version</b>: "1.0"
+<b>summary</b>: Flutter Gallery
+<b>description</b>: |
+    A demo app for Flutter's material design and cupertino widgets, as well
```

Comment:
```suggestion
    A demo app for Flutter's material design [&hellip;]
```

As is just a description we keep it very brief.